src/briefcase/commands/base.py

```diff
@@ -213,6 +215,39 @@ def validate_data_path(self, data_path):
 """
             )
 
+        if not os.path.exists(data_path):
+            try:
+                # The Windows Store version of Python can redirect filesystem
+                # interactions within %LOCALAPPDATA% to a sandboxed location.
+                # To bypass this, the Briefcase cache directory creation is
+                # performed via ``cmd.exe`` in a different process. Once this
+                # directory exists in the "real" %LOCALAPPDATA%, Windows will
+                # allow normal interactions without attempting to sandbox them.
+                if platform.system() == "Windows":
+                    subprocess.run(
+                        ["mkdir", data_path],
+                        shell=True,
+                        check=True,
+                        stdout=subprocess.DEVNULL,
+                        stderr=subprocess.DEVNULL,
+                    )
+                else:
+                    os.makedirs(data_path, exist_ok=True)
+            except (subprocess.CalledProcessError, OSError):
+                raise BriefcaseCommandError(
+                    f"""
+Failed to create the Briefcase directory to store tools and support files:
+
+    {data_path}
+
+Create this directory manually and re-run Briefcase.
```

Comment:
Why this advice? Surely if Briefcase can't create the folder, manually creating it will just mask future permission-related problems?

Reply:
Yeah, it's hard to know why this directory creation would fail in the real world. Right now, if this directory couldn't be created, it would just bomb out. However, I'd expect a knowledgeable user to be able to diagnose and resolve the issue without further problems.

I'd be fine updating the message just to state the problem though and leave it at that. (Although, leaving the note about `BRIEFCASE_HOME` I think makes sense.)

Reply:
Yeah - that would be my inclination as well. Unless we have a reason to believe that the advice will help, it's better to have people come to us with a specific problem, rather than something they've tried to ... fix  :-)

Reply:
Makes sense. Removed remediation recommendations.